package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
```

Comment:
it might be better to keep this script named just `test` - `npm test` is the standard way of invoking tests on a node.js project - we can segregate into `unit`, `e2e`, etc. when we add those additional test categories

@carrotfarmer - thoughts?

Reply:
@floydpink I made the changes in #28